### PR TITLE
fix: able to follow group on event page

### LINKED ISSUE #8091: Groups: Following not possible on event page
It is not possible to follow the Codeheat group. Nothing happens and the console shows errors as below.

Compare https://eventyay.com/e/d932620a

Screenshot from 2021-11-15 23-34-24

> state-machine.js:86 Sentry Logger [Error]: t: An event processor returned null, will not send event.
(anonymous) @ state-machine.js:86
(anonymous) @ state-machine.js:86
a @ state-machine.js:86
e.error @ state-machine.js:86
(anonymous) @ state-machine.js:86
onrejected @ state-machine.js:86
(anonymous) @ state-machine.js:86
_executeHandlers @ state-machine.js:86
_attachHandler @ state-machine.js:86
(anonymous) @ state-machine.js:86
e @ state-machine.js:86
e.then @ state-machine.js:86
e._captureEvent @ state-machine.js:86
e.captureEvent @ state-machine.js:86
e._invokeClient @ state-machine.js:86
e.captureEvent @ state-machine.js:86
callback @ state-machine.js:86
v @ state-machine.js:86
p.onunhandledrejection @ state-machine.js:86
-private.js:5802 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'relationshipMeta')
    at e.i.referenceFor (-private.js:5802)
    at l.belongsTo (-private.js:6811)
    at a.<anonymous> (public.js:189)
    at c (polyfill.js:6473)
    at Generator._invoke (polyfill.js:6702)
    at Generator.e.<computed> [as next] (polyfill.js:6525)
    at F (public.js:25)
    at a (public.js:27)

### PR DESCRIPTION
**Pull request by pc-beast**
Fixes #8091 

#### Checklist

- [ ] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [ ] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/controllers/public.js [hunk 1]
```diff
@@ -80,6 +80,21 @@ export default class PublicController extends Controller {
...
+  }
+
+  @computed
+  get followers() {
+    return this.model.group.get('followers') ? this.model.group.get('followers').toArray() : [];
+  }
+
+  @computed
+  get isFollowed() {
+    return !!this.follower;
```
Comment (progmatic-99) [inline app/controllers/public.js:95]: Why not write `this.follower` ??

**Review by progmatic-99 — CHANGES_REQUESTED** [review]

File: app/controllers/public.js (hunk 1 above)
Comment (pc-beast) [reply app/controllers/public.js:95]: we want the boolean output from here

**Comment by mariobehling** [comment]
Not able to follow group on test deployment. "Unexpected Error"

Compare https://open-event-frontend-git-fork-pc-beast-i8091-eventyay.vercel.app/g/108

Screenshot from 2021-11-23 10-15-07

**Comment by pc-beast** [comment]
> Not able to follow group on test deployment. "Unexpected Error"
> 
> Compare https://open-event-frontend-git-fork-pc-beast-i8091-eventyay.vercel.app/g/108
> 
> Screenshot from 2021-11-23 10-15-07

Opened #8135 as it is unrelated to changes in this PR. This can be merged @mariobehling.

**Review by progmatic-99 — APPROVED** [review]
This can be merged. @mariobehling

**Pull request merged**